src/domain/create-entry.ts

```diff
@@ -30,14 +27,6 @@ export class MetadataParseError extends UserFacingError {
   }
 }
 
-export class AppNotInstalledToForkError extends UserFacingError {
-  public constructor(repository: Repository) {
-    super(
-      `App is not installed to candidate bcr fork ${repository.canonicalName}. You need to configure the app for at least one bazel-central-registry fork. The fork can be in the ruleset's account in the release author's account.`
-    );
-  }
-}
```

Comment:
This error was thrown when attempting to push an entry to the bcr fork. It's no longer needed as we detect this case earlier and the messaging is now in the `NoCandidateForks` error.